src/Cell.cpp

```diff
@@ -59,6 +51,9 @@ Cell::Cell(int id, const char* name) {
   _name = NULL;
   setName(name);
 
+  _num_rings = 0;
+  _num_sectors = 0;
```

Comment:
Well, the ``sectorize`` and ``ringify`` routine actually treat 0 and 1 the same way. If you think 1 is more intuitive, I'll make that change.